### ISSUE TITLE
Contact UP Receivers Regarding Timezone Requirements

### ISSUE THREAD

**Issue by victor-chaparro** [CDCgov/prime-reportstream#14560]
## Problem statement

With proposal complete for handling missing timezone. We want to reach out to existing UP receivers to determine what their requirements and wants are regarding timezone data.

## What you need to know

List all date time fields in HL7

- Is timezone a required field in the above HL7 fields?
- If timezone is not present do you want to receive the message?
- If Yes, would you want us to default to a timezone to ensure messages are valid on your system?
- Should we add a note?
- Sometimes RS receives data without timezone. How do you want to us handle this?

## Acceptance criteria

- [x] Questions asked during IL onboarding
- [ ] Emails sent to all existing UP receivers with above questions

## To do

- [ ] Get list of date time HL7 fields - Victor

**Comment by chris-kuryak** [comment]
Chris to meet with Glenn this week to discuss sending out email to existing UP receivers.

**Comment by chris-kuryak** [comment]
Chris is getting a list of who we need to go back and contact.
Then that will unblock Glenn on sending emails.

**Comment by chris-kuryak** [comment]
List sent to Glenn. Glenn to send emails.

**Comment by chris-kuryak** [comment]
Chris to check with Glenn on responses.

**Comment by chris-kuryak** [comment]
Yes, this is tracked in the Onboarding Tracker as a separate column. The timezone question was asked to existing receivers that onboarded to the UP BEFORE we started asking it to new migrations. The answer are in the spreadsheet.

https://app.smartsheetgov.com/sheets/JRhfQmghpJgGvxG47G4gwV97JpM6jhh62vp7mW91?view=grid

**Comment by chris-kuryak** [comment]
Need to write follow-up ticket to implement those preferences.
